### PR TITLE
NOISSUE: always rebuild backupmanager during slow_tests

### PR DESCRIPTION
**Pull request by afiskon**
Note: no idea what BIN_DIR was used for.

### CODE REVIEW

File: cmd/backupmanager/backupmanager_test.go [hunk 1]
```diff
@@ -38,16 +38,19 @@ import (
...
-		binaryPath = filepath.Join(wd, "..", "..", "bin")
+	if err != nil {
+		panic(err.Error())
+	}
 
-		if err != nil {
-			panic(err.Error())
-		}
+	// Always rebuild backupmanager
+	bashCmd := "cd " + binaryPath + " && (rm backupmanager || true) && go build ../cmd/backupmanager"
```
Comment (dmitry-panchenko) [inline cmd/backupmanager/backupmanager_test.go]: You could use `rm -f` to avoid ugly `|| true`

Comment (afiskon) [reply cmd/backupmanager/backupmanager_test.go]: Nope:
```
$ rm -r nosuchfile
rm: nosuchfile: No such file or directory
$ echo $?
1
```

Comment (afiskon) [reply cmd/backupmanager/backupmanager_test.go]: I believe you are right, tnx.

Comment (afiskon) [reply cmd/backupmanager/backupmanager_test.go]: What I personally dislike about -f is that it may remove a write-protected file.

Comment (dmitry-panchenko) [reply cmd/backupmanager/backupmanager_test.go]: Yeah, but `backupmanager ` could be write-protected?
Your point is fine when you are trying to delete a user input data but in this case you we could ignore "write-protected" part.

**Pull request merged**